scsi-target-utils: to use custom startup script

CGCS has custom systemv startup script.  This patch enables spec file to
to use CGCS custom startup script.  Also puppet packstack expects the
original /etc/tgt/targets.conf file (not from fedora distro), so change
spec file to use the original targets.conf instead.

diff --git a/tmp/scsi-target-utils.spec.org b/SPECS/scsi-target-utils.spec
index 4f134de..5394c3a 100644
--- a/tmp/scsi-target-utils.spec.org
+++ b/SPECS/scsi-target-utils.spec
@@ -26,6 +26,7 @@ Source2:        sysconfig.tgtd
 Source3:        targets.conf
 Source4:        sample.conf
 Source5:        tgtd.conf
+Source6:        tgtd.init
 Patch0:         0001-redhatify-docs.patch
 Patch1:         0002-remove-check-for-xsltproc.patch
 Patch2:         0003-default-config.patch
@@ -99,9 +100,11 @@ Adds support for the Gluster glfs backstore to scsi-target-utils.
 %{__install} -p -m 0644 doc/manpages/tgt-admin.8 %{buildroot}/%{_mandir}/man8
 %{__install} -p -m 0644 doc/manpages/tgt-setup-lun.8 %{buildroot}/%{_mandir}/man8
 %{__install} -p -m 0600 %{SOURCE2} %{buildroot}%{_sysconfdir}/sysconfig/tgtd
-%{__install} -p -m 0600 %{SOURCE3} %{buildroot}%{_sysconfdir}/tgt
+%{__install} -p -m 0600 conf/targets.conf %{buildroot}%{_sysconfdir}/tgt
 %{__install} -p -m 0600 %{SOURCE4} %{buildroot}%{_sysconfdir}/tgt/conf.d
-%{__install} -p -m 0600 %{SOURCE5} %{buildroot}%{_sysconfdir}/tgt
+#%{__install} -p -m 0600 %{SOURCE5} %{buildroot}%{_sysconfdir}/tgt
+mkdir -p %{buildroot}%{_sysconfdir}/rc.d/init.d/
+%{__install} -p -m 0755 %{SOURCE6} %{buildroot}%{_sysconfdir}/rc.d/init.d/tgtd
 
 pushd usr
 %{__make} install %{?with_rdma:ISCSI_RDMA=1} %{?with_rbd:CEPH_RBD=1} %{?with_glfs:GLFS_BD=1} SD_NOTIFY=1 DESTDIR=%{buildroot} sbindir=%{_sbindir} libdir=%{_libdir}/tgt
@@ -132,8 +135,9 @@ pushd usr
 %{_unitdir}/tgtd.service
 %attr(0600,root,root) %config(noreplace) %{_sysconfdir}/sysconfig/tgtd
 %attr(0600,root,root) %config(noreplace) %{_sysconfdir}/tgt/targets.conf
-%attr(0600,root,root) %config(noreplace) %{_sysconfdir}/tgt/tgtd.conf
+#%attr(0600,root,root) %config(noreplace) %{_sysconfdir}/tgt/tgtd.conf
 %attr(0600,root,root) %config(noreplace) %{_sysconfdir}/tgt/conf.d/sample.conf
+%attr(-,root,root) %{_sysconfdir}/rc.d/init.d/tgtd
 
 %if 0%{?with_rbd}
 %files rbd
